modular_skyrat/modules/company_imports/code/armament_datums/nakamura_tools.dm

```diff
@@ -0,0 +1,123 @@
+	category = NAKAMURA_ENGINEERING_TOOLING_NAME
+	company_bitflag = CARGO_COMPANY_NAKAMURA_TOOLING
+
+// Basic, non-power tools, as well as other related equipment
+
+/datum/armament_entry/company_import/nakamura_tooling/basic
+	subcategory = "Standard Engineering Equipment"
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/analyzer
+	item_type = /obj/item/analyzer
+	lower_cost = CARGO_CRATE_VALUE * 0.1
+	upper_cost = CARGO_CRATE_VALUE * 0.6
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/mechanical_toolbox
+	item_type = /obj/item/storage/toolbox/mechanical
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/electrical_toolbox
+	item_type = /obj/item/storage/toolbox/electrical
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/multitool
+	item_type = /obj/item/multitool
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/inducer
+	item_type = /obj/item/inducer
+	lower_cost = CARGO_CRATE_VALUE * 0.25
+	upper_cost = CARGO_CRATE_VALUE
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/magboots
+	item_type = /obj/item/clothing/shoes/magboots
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/basic/insuls
+	item_type = /obj/item/clothing/gloves/color/yellow
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+// Advanced stuff like power tools and holofans
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced
+	subcategory = "Advanced Engineering Equipment"
+	interest_required = COMPANY_SOME_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/skillchip
+	item_type = /obj/item/skillchip/job/engineer
+	lower_cost = CARGO_CRATE_VALUE * 0.75
+	upper_cost = CARGO_CRATE_VALUE * 1.25
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/ranged_analyzer
+	item_type = /obj/item/analyzer/ranged
+	lower_cost = CARGO_CRATE_VALUE * 0.75
+	upper_cost = CARGO_CRATE_VALUE * 1.25
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/forcefield
+	item_type = /obj/item/forcefield_projector
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/atmos_fan
+	item_type = /obj/item/holosign_creator/atmos
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/powerdrill
+	item_type = /obj/item/screwdriver/power
+	lower_cost = CARGO_CRATE_VALUE * 1.5
+	upper_cost = CARGO_CRATE_VALUE * 3
+
+/datum/armament_entry/company_import/nakamura_tooling/advanced/jaws
+	item_type = /obj/item/crowbar/power
```

Comment:
(idk if im allowed to comment in here kick me out if so, but) as an engineering player having the ability to get jaws in the unlikely event that science goes AWOL would be nice since they're kinda a critical item - and if it requires some RP and getting forms stamped to send out a cargo order, then even better.

Reply:
That's why I left them in but restricted, the CE can still order some and you can still get cargo to let you order some, restricting them just stops joe assistant from walking up to the requests console and ordering one.